src/platform/forms-system/src/js/definitions/autosuggest.js

```diff
@@ -26,11 +27,12 @@ export const schema = {
  */
 export function uiSchema(label, getOptions, options = {}) {
   const validations = [];
-  if (!_.get('ui:options.freeInput', options)) {
+  if (!get('ui:options.freeInput', options)) {
```

Comment:
```suggestion
  if (!options['ui:options'].freeInput') {
  ```

Reply:
Yeah, since we're only doing a check on the `options` object, this should work.